### PR TITLE
fix: update aws provider version

### PR DESCRIPTION
**Pull request by tirumerla**
## what
* Update aws provider to `>= 2.0` avoid module version conflicts when used with `>= 3.0` modules (eg. `aws-eks-node-group`).

## why
* This module is being used with [terraform-aws-eks-node-group](https://github.com/cloudposse/terraform-aws-eks-node-group) which needs aws provider version to be `>= 3.0` which is causing version conflicts.

## references
* See cloudposse/terraform-aws-eks-node-group#41 for details. This tackles one of the modules causing the issue.

### CODE REVIEW

**Comment by tirumerla** [comment]
Oops, didn't realize there was #53. Feel free to pick one and close the other.

File: versions.tf [hunk 1]
```diff
@@ -2,7 +2,7 @@ terraform {
   required_version = ">= 0.12.0, < 0.14.0"
```
Comment (aknysh) [inline versions.tf:2]:
```suggestion
  required_version = ">= 0.12.0"
```

File: versions.tf [hunk 2]
```diff
@@ -2,7 +2,7 @@ terraform {
...
 
   required_providers {
-    aws   = "~> 2.0"
+    aws   = ">= 2.0"
     tls   = "~> 2.0"
```
Comment (aknysh) [inline versions.tf:6]:
```suggestion
    tls   = ">= 2.0"
```

File: versions.tf [hunk 3]
```diff
@@ -2,7 +2,7 @@ terraform {
...
     local = "~> 1.3"
```
Comment (aknysh) [inline versions.tf:7]:
```suggestion
    local = ">= 1.3"
```

File: versions.tf [hunk 4]
```diff
@@ -2,7 +2,7 @@ terraform {
...
     null  = "~> 2.1"
```
Comment (aknysh) [inline versions.tf:8]:
```suggestion
    null  = ">= 2.1"
```

**Review by aknysh — COMMENTED** [review]
thanks @tirumerla  please see comments

**Review by aknysh — APPROVED** [review]

**Pull request merged**